src/components/CallToAction/index.js

```diff
@@ -30,41 +35,57 @@ const secondary = cntl`
 `
 
 const outline = cntl`
-    bg-tan/75
+    bg-tan
+    bg-opacity-75
     dark:bg-primary
     text-primary
     text-opacity-80
     hover:text-opacity-100
     dark:text-primary-dark
     hover:text-primary
+    border-opacity-10
     hover:border-opacity-25
     active:border-opacity-50
-    border-primary/10
+    border-primary
     dark:border-primary-dark
 `
 
 const buttonTypes = {
     primary,
     secondary,
     outline,
+    custom: '',
 }
 
 const button = (type = 'primary', width = 'auto', className = '', size = 'lg') => cntl`
     text-center
     select-none
-    rounded-full
+    rounded-sm
     inline-block
     cta
+    relative
+    active:top-[1px]
+    active:scale-[.97]
     w-${width}
     ${buttonTypes[type] || ''}
     ${sizes[size]}
     ${className}
 `
 
-export const TrackedCTA = ({ event: { name: eventName, ...event }, ...props }) => {
+export const TrackedCTA = ({ event: { name: eventName, ...event }, featureFlag, ...props }) => {
     const { posthog } = useValues(posthogAnalyticsLogic)
 
-    return <CallToAction {...props} onClick={() => posthog?.capture(eventName, event)} />
+    const handleClick = () => {
+        posthog?.capture(eventName, event)
```

Comment:
I assume this is the main event that the experiment is tracking?

If so, you're missing the property (step 2 of tutorial). This should be something like:

```suggestion
        posthog?.capture(eventName, {...event, [`$feature\{feature_flag}`]: featureFlag.response})
```

---

If I'm mistaken and this isn't the main event we're tracking, could you point me to it?

Reply:
This is where the main event that should be associated with the feature flag gets called, yes! However, I'm tracking multiple events (CTA clicks for PostHog deployment) on this branch that should be associated with the feature flag. I might be doing that the wrong way, though.

Reply:
Worth testing on netlify deploy that when this event is called, there's a `$feature/<feature_name>` property in PostHog, then you're golden!